src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  return (valueX) => {
+    callCount++;
+
+    return (valueY) => {
+      if (callCount > 3 && callCount % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return valueX + valueY;
```

Comment:
You can use here ternary operator, it will be much shorter
`return <condition> ? 'Bzz...' : 'x + y'`